test/test.js

```diff
@@ -473,6 +491,23 @@ describe('YeldaChat', () => {
   })
 
   describe('yeldaChat.resetChat', () => {
+
+    it('should load data of the second call', async () => {
+      yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'fr_FR' }))
+      await yeldaChat.resetChat(Object.assign({}, validMockData, { locale: 'en_US' }))
+
+      expect(yeldaChat.locale).to.equal('en_US')
+    })
+
+    it('should block loadChat if the data do not match', async () => {
```

Comment:
```suggestion
    it('should not set webChatContainer if loadChat  locale parameter does not match global locale', async () => {
```
would be closer to reality too